### DISCUSSION TITLE
Säännöllinen huoltoikkuna 12.7.2023

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#690] [Huolto- ja häiriötiedotteet]
Heinäkuun säännöllisessä huoltoikkunassa tehdään pieni muutos palomuurille ja se aiheuttaa hetkellisen käyttökatkoksen. Muutos tehdään klo 7-8 välillä.

**Comment by lmstrand** [comment]
Muutos tehty 12.7. klo 7.02.
